compiler/rustc_error_messages/locales/en-US/middle.ftl

```diff
@@ -0,0 +1,17 @@
+middle_drop_check_overflow =
+    overflow while adding drop-check rules for {$ty}
+    .note = {$note}
```

Comment:
```suggestion
    .note = overflowed on {$overflow_ty}
```

Why not this?

Reply:
Sorry David, I haven't got much time to work on it lately. You can merge this one and unassign me from this crate. I will come back and pick up anything left when I got time.